src/utils/Validation.js

```
@@ -0,0 +1,73 @@
+     * @param {string} address - The wallet address to validate.
+     * @returns {boolean} Returns true if the address is valid; otherwise, false.
+     */
+    static isWalletAddress(address) {
+        if (!address || typeof address !== 'string') {
+            return false;
+        }
+
+        try {
+            return ADDRESS_REGEX.test(address) && isValidAddress(address);
+        } catch (error) {
+            logger.error(`${this.name}: Error occurred during address validation.`, error);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)